assembl/views/discussion/views.py

```diff
@@ -175,25 +175,28 @@ def home_view(request):
     return response
 
 
+@view_config(route_name='new_home', request_method='GET', http_cache=60,
+             renderer='assembl:templates/index_react.jinja2')
+@view_config(route_name='new_styleguide', request_method='GET',
+             renderer='assembl:templates/index_react.jinja2')
+@view_config(route_name='new_profile', request_method='GET',
+             renderer='assembl:templates/index_react.jinja2')
```

Comment:
It's too bad to create routes in the backend and in the frontend. If I need to create ./v2/whatever, I have to do it in the backend also.
To resume, there is 2 cases: I need a discussion ID or I don't need a discussion ID. Is it possible to find a solution with this approach?